crates/metadata-store/src/local/service.rs

```diff
@@ -62,40 +66,38 @@ impl LocalMetadataStoreService {
         let bind_address = options.bind_address.clone();
         let store = LocalMetadataStore::create(options, rocksdb_options).await?;
         // Trace layer
-        let span_factory = tower_http_0_4::trace::DefaultMakeSpan::new()
+        let span_factory = tower_http::trace::DefaultMakeSpan::new()
             .include_headers(true)
             .level(tracing::Level::ERROR);
 
-        let reflection_service_builder = tonic_reflection_0_10::server::Builder::configure()
+        let reflection_service_builder = tonic_reflection::server::Builder::configure()
```

Comment:
beware that latest tonic_reflection was updated to grpc reflection `v1` imports, perhaps double check if it still works with `grpcurl`/`grpcui`

Reply:
I haven't tried this one but I wouldn't be too worried because the metadata store and the node server are not really user facing.

Reply:
It seems to work with my grpcurl version `1.8.9`.